checkers_bot_tournament/bots/base_bot.py

```diff
@@ -8,7 +8,22 @@
 class Bot(ABC):
     def __init__(self, bot_id: int) -> None:
         self.bot_id = bot_id
+        self.move_history: list[Move] = []
 
+    def play_move_wrapper(self, board: Board, colour: Colour, move_list: list[Move]) -> int:
```

Comment:
move history doesn't reset across different games

suggest if previous_move is None, then that must've been the first move -> reset list